webapp/src/main/java/com/box/l10n/mojito/entity/Branch.java

```diff
@@ -31,11 +32,23 @@
           unique = true),
       @Index(name = "I__BRANCH__DELETED", columnList = "deleted")
     })
+@NamedEntityGraph(
+    name = "Branch.legacy",
+    attributeNodes = {
+      @NamedAttributeNode(value = "notifiers"),
+      @NamedAttributeNode(value = "screenshots", subgraph = "Branch.legacy.screenshots"),
+    },
```

Comment:
We're missing the fetching of `createdByUser` here, if missing it causes the branch notifier failures?